Amplitude.podspec

```diff
@@ -26,4 +26,6 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target  = '3.0'
   s.watchos.source_files       = 'Sources/Amplitude/**/*.{h,m}'
   s.watchos.resources          = 'Sources/Amplitude/**/*.{der}'
+
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
```

Comment:
Small nit: Could this be moved to line 13

Reply:
moved!